tests/phpunit/api/v3/CustomFieldTest.php

```diff
@@ -569,4 +569,31 @@ public function getCustomFieldKeys($getFieldsResult) {
     return $r;
   }
 
+  /**
+   * This test is designed to ensure that when a custom field is updated via the
+   * API, params that are not supplied do not revert to the defaults. This was
+   * happening with, for example, is_searchable
+   */
+  public function testDisableSearchableContactReferenceField() {
+    $customGroup = $this->customGroupCreate(array(
+      'name' => 'testCustomGroup',
+      'title' => 'testCustomGroup',
+      'extends' => 'Individual',
+    ));
+    $params = array(
+      'name' => 'testCustomField',
+      'label' => 'testCustomField',
+      'custom_group_id' => 'testCustomGroup',
+      'data_type' => 'ContactReference',
+      'html_type' => 'Autocomplete-Select',
+      'is_searchable' => '1',
+    );
+    $result = $this->callAPISuccess('CustomField', 'create', $params);
+    $params = [
+      'id' => $result['id'],
+      'is_active' => 0,
+    ];
+    $result = $this->callAPISuccess('CustomField', 'create', $params);
```

Comment:
@michaelmcandrew looks like your not actually asserting anything other than the API works

What you probably want do to is add in `$this->assertTrue($result['values'][$result['id']]['is_searchable'])`

Reply:
@seamuslee001 - The test failed with an exception before the  patch not applied. Just seeing it work was good enough for me. Though I take your point that I could be testing a bit more with the test.